stdlib/lazy.mli

```diff
@@ -67,6 +68,20 @@ external force : 'a t -> 'a = "%lazy_force"
    recursively.
 *)
 
+val is_val : 'a t -> bool
+(** [is_val x] returns [true] if [x] has already been forced and
+    did not raise an exception.
+    @since 4.00.0 *)
+
+val map : ('a -> 'b) -> 'a t -> 'b t
+(** [map f x] returns a suspension that, when forced,
+    forces [x] and applies [f] to its value. *)
```

Comment:
Missing "since" line.

Reply:
Indeed! Fixed, thanks.

(I wish we had an automated check for those.)